frontend/pubspec.lock

```diff
@@ -321,7 +321,7 @@ packages:
       name: plugin_platform_interface
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.1.2"
+    version: "2.1.3"
   provider:
     dependency: "direct main"
     description:
```

Comment:
Nie dotyczy tego konkretnego PR, ale dobrze by było zbadać, czy `pubspec.lock` można wrzucić do `.gitignore` i czy to jest bezpieczne